flux/src/test/java/software/amazon/aws/clients/swf/flux/wf/graph/WorkflowGraphBuilderTest.java

```diff
@@ -1441,6 +1442,26 @@ public void build_DoesValidateAttributeInputsIfInitialInputAttributesSpecified_A
         Assert.assertNotNull(builder.build());
     }
 
+    @Test
+    public void build_DoesValidateAttributeInputsIfInitialInputAttributesSpecified_AllowsStartTimeAttributesAsDate() {
+        WorkflowStep start = new TestStepExpectsStartTimeAttributesAsDate();
+
+        WorkflowGraphBuilder builder = new WorkflowGraphBuilder(start, Collections.emptyMap());
+        builder.alwaysClose(start);
+
+        Assert.assertNotNull(builder.build());
+    }
+
+    @Test
+    public void build_DoesValidateAttributeInputsIfInitialInputAttributesSpecified_AllowsStartTimeAttributesAsInstant() {
+        WorkflowStep start = new TestStepExpectsStartTimeAttributesAsDate();
```

Comment:
you meant for this to be the instant class, not the Date class.

Reply:
good catch, thanks.